pkgs/development/libraries/rocfft/default.nix

```diff
@@ -16,138 +16,40 @@
 , fftwFloat
 , buildTests ? false
 , buildBenchmarks ? false
+, gpuTargets ? [
```

Comment:
I suggest using hip.gpuTargets and adding the xnacks.

Reply:
Oh, I was originally going to, but there was [a comment](https://github.com/NixOS/nixpkgs/blob/f4aa6afa5f934ece2d1eb3157e392d056be01617/pkgs/development/compilers/hip/default.nix#L165-L166) mentioning that we can't use it for all projects. Do we still want to do that anyway? I was thinking that we'd just have to update this list on every update.

Reply:
Actually you're right, and it makes sense to use them custom this time.
Normally you don't have to specify them manually since the default is to build all, but since we're doing other stuff anyway this is fine.